services/src/main/java/za/co/sfy/lucid/sams/rest/vo/data/writer/SubjectCriteriaRequest.java

```diff
@@ -0,0 +1,329 @@
+
+package za.co.sfy.lucid.sams.rest.vo.data.writer;
+
+import java.io.Serializable;
+import java.sql.Date;
+
+
+
+public class SubjectCriteriaRequest implements Serializable {
+
+    private static final long serialVersionUID = 6627252016141632712L;
+    
+    private Integer subjectID;
+    private Integer criterionID;
+    private Short description;
+    private Integer grade;
+    private Integer weighting;
+    private Short subjectlevel;
+    private Integer criterionscore;
+    private Short datayear;
+    private Short subheading;
+    private Date dateadded;
+    private Short type;
+    private Long outcomes;
+    private Long activities;
+    private Long assessments;
+    private Integer sectionID;
+    private Boolean useactivities;
+    private Boolean includeffl;
+    private Boolean includeExam;
+    private Short updated;
+    private Boolean quaterlytest;
+    private Boolean fetcommontest;
+    private Short descriptionafr;
+    private Short descriptionvern;
```

Comment:
Consider replacing Short with Integer, if request work with short then you dont have to